lib/gds_api/test_helpers/email_alert_api.rb

```diff
@@ -472,6 +472,27 @@ def stub_email_alert_api_bulk_unsubscribe_bad_request_with_message(slug:, govuk_
         ).to_return(status: 422)
       end
 
+      def stub_update_subscriber_list_details(slug:, params:)
+        stub_request(:patch, "#{EMAIL_ALERT_API_ENDPOINT}/subscriber-lists/#{slug}")
+        .with(body: params.to_json)
+        .to_return(
+          status: 200,
+          body: get_subscriber_list_response(params).to_json,
+        )
+      end
+
+      def stub_update_subscriber_list_details_not_found(slug:, params:)
+        stub_request(:patch, "#{EMAIL_ALERT_API_ENDPOINT}/subscriber-lists/#{slug}")
+        .with(body: params.to_json)
+        .to_return(status: 404)
+      end
+
+      def stub_update_subscriber_list_details_unprocessible_entity(slug:, params: {})
+        stub_request(:patch, "#{EMAIL_ALERT_API_ENDPOINT}/subscriber-lists/#{slug}")
+        .with(body: params.to_json)
+        .to_return(status: 422)
+      end
```

Comment:
These should all have the same default `params` value.

Reply:
should they?
For not found, you must have got past the 422 errors, so you will have needed to provide something as params on 484?
Same for a sucessful case.

the stub on 490 is a bit different, you get a 422 from an empty set of body params, but also from invalid ones. I figure as the dev using the stub is choosing to stick this in, they can choose if they want params or not?

isn't it better we return an `ArgumentError` for those first two cases if you don't give params?